.github/workflows/ci.yml

```diff
@@ -8,24 +8,16 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node-version:
-        - 12
-        npm-test:
-        - i18n_extract
-        - is-es5
-        - lint
-        - test
     steps:
     - uses: actions/checkout@v2
-    - uses: actions/setup-node@v2
+    - run: echo "NODE_VER=`cat .nvmrc`" >> $GITHUB_ENV
+    - name: setup node
+      uses: dcodeIO/setup-node-nvm@master
```

Comment:
nit: might not really be an issue but does this Github Action support specifying a version so it doesn't run off of `master`?

Reply:
good flag! I think it would be great to pin this rather than use master. And be changing versions as needed. looks like we can based on their own doc 

```
- name: Set up node using nvm
  uses: dcodeIO/setup-node-nvm@v4
```

Reply:
using v4 now